lib/kubernetes-deploy/runner.rb

```diff
@@ -295,25 +296,38 @@ def validate_configuration
       KubernetesDeploy.logger.info("All required parameters and files are present")
     end
 
-    def update_tprs(resources)
-      resources.each do |r|
+    def deploy_resources(resources, prune: false)
+      KubernetesDeploy.logger.info("Deploying resources:")
+
+      # Apply can be done in one large batch, the rest have to be done individually
+      applyables, individuals = resources.partition { |r| r.deploy_method == :apply }
+
+      individuals.each do |r|
         KubernetesDeploy.logger.info("- #{r.id}")
         r.deploy_started = Time.now.utc
-        _, _, st = run_kubectl("replace", "-f", r.file.path)
+        case r.deploy_method
+        when :replace
+          _, _, st = run_kubectl("replace", "-f", r.file.path)
+        when :replace_force
+          _, _, st = run_kubectl("replace", "--force", "-f", r.file.path)
+        end
```

Comment:
Can you add an `else` case that raises? Shouldn't ever get hit, but could be helpful for debugging a future stupid mistake.